docs/status.md

```diff
@@ -133,7 +133,7 @@ one of the following tracking labels.
 | rsqrt                    | yes           | yes          | yes            | yes             | no          |
 | scatter                  | yes           | revisit      | no             | no              | no          |
 | select                   | yes           | yes          | yes            | yes             | no          |
-| select_and_scatter       | no            | revisit      | no             | no              | no          |
+| select_and_scatter       | no            | revisit      | yes*           | no              | no          |
```

Comment:
"Type Inference" should be set to "yes" because we now have a spec for `select_and_scatter`, and this implementation satisfies it.